constellations/gemini-admin/src/ask_for_help/afhEdit.js

```diff
@@ -0,0 +1,234 @@
+            (this.state.isEdit ? "Edit" : "Add") + " Ask For Help Session";
+
+        const optLocations = this.state.locations.map((loc, index) => (
+            <option key={index}>{loc.locationId}</option>
+        ));
+
+        let deleteButton = <div></div>;
+        if (this.state.isEdit) {
+            deleteButton = (
+                <button className="btn-delete" onClick={this.onClickDelete}>
+                    Delete
+                </button>
+            );
+        }
+
+        let modalDiv;
+        let modalContent;
+        let showModal;
+        if (this.state.showDeleteModal) {
+            showModal = this.state.showDeleteModal;
+            modalContent = (
+                <YesNoModal
+                    text={"Are you sure you want to delete?"}
+                    onAccept={this.onConfirmDelete}
+                    onReject={this.onDismissModal}
+                />
+            );
+        }
+        if (this.state.showSaveModal) {
+            showModal = this.state.showSaveModal;
+            modalContent = (
+                <OkayModal
+                    text={"Ask for help information saved!"}
+                    onOkay={this.onSavedOk}
+                />
+            );
+        }
+        if (modalContent) {
+            modalDiv = (
+                <Modal
+                    content={modalContent}
+                    show={showModal}
+                    onDismiss={this.onDismissModal}
+                />
+            );
+        }
+
+        return (
+            <div id="view-afh-edit">
+                {modalDiv}
+                <h2>{title}</h2>
+
+                <AFHEditDateTime
+                    postedAt={this.state.date}
+                    onMomentChange={this.onMomentChange}
+                />
+
+                <h3>Time</h3>
+                <input
+                    value={this.state.timeString}
+                    onChange={(e) => this.handleChange(e, "timeString")}
+                />
+
+                <h3>Title</h3>
+                <input
+                    value={this.state.title}
+                    onChange={(e) => this.handleChange(e, "title")}
+                />
+
+                <h3>Subject</h3>
+                <select
+                    value={this.state.subject}
+                    onChange={(e) => this.handleChange(e, "subject")}>
+                    <option>Math</option>
+                    <option>English</option>
+                    <option>Computer Programming</option>
+                </select>
+
+                <h3>Location ID</h3>
+                <select
+                    value={this.state.locationId}
+                    onChange={(e) => this.handleChange(e, "locationId")}>
+                    {optLocations}
+                </select>
+
+                <h3>Notes</h3>
+                <textarea
+                    value={this.state.notes}
+                    onChange={(e) => this.handleChange(e, "notes")}
+                />
```

Comment:
There's also an isTextBox parameter for the InputText component which you can use to have a textarea instead of an input element.